extensions/skyportal/static/js/components/ProfileDropdown.css

```diff
@@ -0,0 +1,85 @@
+.profileDesktop {
```

Comment:
Why is this file duplicated?

Reply:
The background color! It's very subjective, but I think that SP's corporate light blue color does not match the side bar and everyone liked Josh's original Figma template, including the rough color scheme. So that's my way of overriding the default colors :) The same goes for `SidebarAndHeader.jsx`. 
What is the proper way of overriding/styling?

Reply:
You're right---there's no straightforward way to override that.  Unfortunately, overwriting the CSS file may well stop working at any point when we switched to styled components or similar.

Why don't we just make this the default in SkyPortal for now; I don't mind.